src/app/components/utility/side-bar/side-bar.component.scss

```diff
@@ -16,8 +14,10 @@
 }
 
 .links-section-outer {
+  border-right: 1px solid $grad-light-black;
   overflow: auto;
-  height: 100%;
+  height: calc(100% - 132px);
```

Comment:
You should change it to `calc(100% - 120px);`

Reply:
> You should change it to `calc(100% - 120px);`

O ok..I'll do it now